track/framebuffer.py

```diff
@@ -68,6 +122,25 @@ def generate_optical_flow(self, opt_flow, flow_threshold=40):
             scaled_flow = np.clip(flow * 256, -16000, 16000)
             self.flow.append(scaled_flow)
 
+    def get_frame(self, frame_number):
+        if self.cache:
+            frame = self.cache.get_frame(frame_number)
```

Comment:
it's more Pythonic and makes the method easier to read if the frame is just returned here